src/components/victory-group/victory-group.js

```diff
@@ -404,6 +411,7 @@ export default class VictoryGroup extends React.Component {
         theme: child.props.theme || props.theme,
         labelComponent: props.labelComponent || child.props.labelComponent,
         style,
+        domainPadding: {x: (props.width / noOfChildren) / 2},
```

Comment:
ah okay this chart has an offset set. but maybe that's just something the user has to do. makes more sense than setting the domain.